docker-entrypoint.sh

```diff
@@ -19,6 +19,9 @@ CUSTOM_ARGS=""
 if [ -n "$TELEGRAM_STAT" ]; then
   CUSTOM_ARGS="${CUSTOM_ARGS} --http-stat-port=8082"
 fi
+if [ -n "$TELEGRAM_STAT_HIDE_SENSIBLE_DATA" ]; then
```

Comment:
"stats" or "stat"?

Reply:
I'd say stats, but the other variable already is `$TELEGRAM_STAT`, not stats.

Reply:
Uh, ok